template-parts/rss/concise.php

```diff
@@ -0,0 +1,39 @@
+<?php
+header( 'Content-Type: ' . feed_content_type( 'rss2' ) . '; charset=' . get_option( 'blog_charset' ), true );
+echo '<?xml version="1.0" encoding="' . get_option( 'blog_charset' ) . '"?' . '>';
+?>
+<rss version="2.0"
+	xmlns:content="http://purl.org/rss/1.0/modules/content/"
+	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
+	xmlns:dc="http://purl.org/dc/elements/1.1/"
+	xmlns:atom="http://www.w3.org/2005/Atom"
+	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
+	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
+	<?php do_action( 'rss2_ns' ); ?>>
+	<channel>
+		<title><?php bloginfo_rss( 'name' ); ?> - Feed</title>
+		<atom:link href="<?php self_link(); ?>" rel="self" type="application/rss+xml" />
+		<link><?php bloginfo_rss( 'url' ) ?></link>
+		<description><?php bloginfo_rss( 'description' ) ?></description>
+		<lastBuildDate><?php echo mysql2date( 'D, d M Y H:i:s +0000', get_lastpostmodified( 'GMT' ), false ); ?></lastBuildDate>
+		<language><?php echo bloginfo_rss( 'language' ); ?></language>
+		<sy:updatePeriod><?php echo apply_filters( 'rss_update_period', 'hourly' ); ?></sy:updatePeriod>
+		<sy:updateFrequency><?php echo apply_filters( 'rss_update_frequency', '1' ); ?></sy:updateFrequency>
+		<?php do_action( 'rss2_head' ); ?>
+		<?php 
+		while( have_posts() ) : the_post();
+		?>
+			<item>
+				<title><?php the_title_rss(); ?></title>
+				<link><?php the_permalink_rss(); ?></link>
+				<pubDate><?php echo mysql2date( 'D, d M Y H:i:s +0000', get_post_time( 'Y-m-d H:i:s', true ), false ); ?></pubDate>
+				<dc:creator></dc:creator>
+				<guid isPermaLink="false"><?php the_guid(); ?></guid>
+				<description><![CDATA[<?php echo the_excerpt_rss(); ?>]]></description>
```

Comment:
`the_excerpt_rss()` should not be echo'ed here (similar to other functions in the template, like `the_title_rss()`/`the_permalink_rss()`